server/src/services/classroom.ts

```diff
@@ -0,0 +1,55 @@
+import { prisma } from "../utils/prisma.db";
+
+const create = async (userId: number, name: string, capacity: number) => {
+  const classroom = await prisma.classroom.create({
+    data: {
+      userId,
+      name,
+      capacity,
+    },
+  });
+
+  return classroom;
+};
+
+const getById = async (id: number) => {
+  const classroom = await prisma.classroom.findUniqueOrThrow({
+    where: { id },
+  });
+
+  return classroom;
+};
+
+const getAll = async () => {
+  const classroom = await prisma.classroom.findMany({});
```

Comment:
Is the empty object necessary?

Reply:
It is not. I was testing selecting, filtering, etc. and I forgot to remove it.